blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -169,6 +169,9 @@ Changes
 Fixes
 =====
 
+- Fixed an issue that would cause the window functions to yield incorrect
+  results if executed against partitioned windows (ordered or unordered).
```

Comment:
Would it make sense to merge this with the other 2 fixes entries related to window functions? Or at least group them together.

Reply:
good shout. grouped them together (I think there's some value in having them separated)